packages/toolpad-app/cli/index.ts

```diff
@@ -21,7 +21,7 @@ async function runCommand(cmd: 'dev' | 'start', { dir, ...args }: Omit<RunOption
     cmd,
   });
 
-  process.on('SIGINT', () => {
+  process.once('SIGINT', () => {
     app.dispose().then(() => {
```

Comment:
Looks like I didn't push this after all. Let's still do:
```suggestion
    app.dispose().finally(() => {
```